app/handlers/routes.py

```diff
@@ -0,0 +1,140 @@
+import json
+
+from app.db.fb import Firebase
+import barreleye
+
+from flask import Response, request
+
+
+firebase = Firebase()
+
+
+def configure_routes(app):
+    @app.route("/ping", methods=["GET"])
+    def ping():
+        return "PONG"
+
+    @app.route("/", methods=["GET"])
+    def root():
+        return "lubdub"
+
+    @app.route("/booze", methods=["GET"])
+    def booze():
+        result = firebase.fb.get("/", None)
+        return Response(response=json.dumps(result), mimetype="application/json")
+
+    @app.route("/drinks", methods=["GET"])
```

Comment:
@sadams This is looking awesome!  One little thing, it looks like something is up the `/drinks` endpoint.  I'm getting a wild error:
`127.0.0.1 - - [17/Aug/2021 11:44:52] "÷óÌjk±Û
                                             ½9!AûzÞ¿óñ£)¢÷- -¶ì[@k+¬@IoDÅ¿uBÈÊFáÀê³/À+À0À,Ì©Ì¨À        ÀÀ" HTTPStatus.BAD_REQUEST -`
                                             
Currently looking for what might be causing this issue!  All other endpoints are working great though!

Reply:
@sadams actually ignore me! I "think" this issue is being caused by us trying to hit an `https://` endpoint from a local `http://` server!